From f5c0021a016116c53d21d8d5902ea92e682658b6 Mon Sep 17 00:00:00 2001
From: Stacy Gaikovaia <Stacy.Gaikovaia@windriver.com>
Date: Wed, 16 Sep 2020 13:45:07 -0400
Subject: [PATCH] memcheck vgtests remove fullpath-after flags

Test executables produced when cross-compiling can contain
relative paths containing version number, such as:
    coregrind/tests/../../../valgrind-3.16.1/coregrind

Remove the --fullpath-after option so yocto project doesn't
have to upgrade patch every valgrind uprev. Upgrade test stderr
paths in corresponding tests .bb script.

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Stacy Gaikovaia <Stacy.Gaikovaia@windriver.com>
---
 memcheck/tests/badfree3.vgtest | 2 +-
 memcheck/tests/varinfo5.vgtest | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/memcheck/tests/badfree3.vgtest b/memcheck/tests/badfree3.vgtest
index 3dfc5fd..4ee03f0 100644
--- a/memcheck/tests/badfree3.vgtest
+++ b/memcheck/tests/badfree3.vgtest
@@ -1,3 +1,3 @@
 prog: badfree
-vgopts: -q --fullpath-after=memcheck/ --fullpath-after=coregrind/
+vgopts: -q
 stderr_filter_args: badfree.c
diff --git a/memcheck/tests/varinfo5.vgtest b/memcheck/tests/varinfo5.vgtest
index 063d00d..79c4a72 100644
--- a/memcheck/tests/varinfo5.vgtest
+++ b/memcheck/tests/varinfo5.vgtest
@@ -1,3 +1,3 @@
 prog: varinfo5
-vgopts: --fullpath-after=memcheck/  --fullpath-after=coregrind/ --read-var-info=yes --read-inline-info=yes -q
+vgopts: --read-var-info=yes --read-inline-info=yes -q
 stderr_filter: filter_varinfo3
